### ISSUE TITLE
Update text color for "Data updated on... " in Stakeholder Details

### ISSUE THREAD

**Issue by andyarensman** [hackforla/food-oasis#2109]
### Overview
In the Stakeholder Details side panel, the orange "Data updated on... " text should be the same color as the rest of the text, not orange.

![chrome_1CROr2HypI](https://github.com/hackforla/food-oasis/assets/86622025/141fe1bb-e3e3-45fb-87b8-2e1ffb2f0c0c)

### Action Items
- [ ] update the color of the "Data updated on... " text to the same as the other text (#313233). Note: not every location has this text.
